runtime-image/jdk11-xp-overrides.yaml

```diff
@@ -0,0 +1,28 @@
+schema_version: 1
```

Comment:
@luck3y would it be possible to change the layout to have everything related to eap-xp under a eap-xp directory instead of mixing that with EAP 7.3 images.

Something like:

```
eap-xp/
   image.yaml
   container.yaml
   rel-jdk11-overrides.yaml 
   runtime-image/
     jdk8-overrides.yaml
     jdk11-overrides.yaml
```

Reply:
@jmesnil Yep, I'll switch it over to this.

Reply:
I've update this below, I'll squash if it looks OK.